pkg/cloudprovider/aws/capacity.go

```diff
@@ -58,7 +62,8 @@ func (c *Capacity) Create(ctx context.Context, constraints *cloudprovider.Constr
 	var instanceIds []*string
 	podsForInstance := make(map[string][]*v1.Pod)
 	for _, packing := range instancePackings {
-		instanceID, err := c.instanceProvider.Create(ctx, launchTemplate, packing.InstanceTypes, zonalSubnetOptions)
+		instanceID, err := c.instanceProvider.Create(ctx, launchTemplate,
```

Comment:
I'm always a bit wary of patterns like

```myObject.Method1(foo, myObject.Method2(bar))```
It typically means that Method1 should just encapsulate the concepts of Method2. There are, of course, cases where that doesn't make sense. I think it does here, though.